content/blog/typescript-type-generics/index.md

```diff
@@ -105,16 +105,16 @@ function returnSelf(returnProp: any): any {
 const returnedObject = returnSelf({objProperty: 12}); // This now works! 🎉
 
 returnedObject.test(); // This will not return an error but should 🙁
-returnedObject.objProperty; // This will also (correctly) not throw an error, but TS will not know it's a number ☹️
+console.log(returnedObject.objProperty); // This will also (correctly) not throw an error, but TS will not know it's a number 🙁
```

Comment:
I'd like to revert that. I know TS is complaining but that's also sorta the point 😅Having the `returnedObj` on different starting indexes makes this harder to scan

```suggestion
returnedObject.objProperty; // This will also (correctly) not throw an error, but TS will not know it's a number ☹️
```

I also intentionally made the emoji more sad 😜

Reply:
Ok yeah I get that for the `console.log` thing. The emoji was just a bit confusing since at least on linux in FF it only appears in monochrome even though the other one gets color.